packages/react-native-web/src/exports/TextInput/index.js

```diff
@@ -401,7 +401,7 @@ class TextInput extends Component<*> {
         onSubmitEditing(e);
       }
       if (shouldBlurOnSubmit) {
-        this.blur();
+        if (this.blur) this.blur();
```

Comment:
The tests were failing with `_this.blur is not a function`, not sure what's the best way to fix is, so I changed this code.